Runtime/Tracking/Velocity/ConstantVelocityTracker.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace Zinnia.Tracking.Velocity
+{
+    using Malimbe.PropertySerializationAttribute;
+    using Malimbe.XmlDocumentationAttribute;
+    using UnityEngine;
+
+    /// <summary>
+    /// Provides the velocity as set on its properties.
+    /// </summary>
+    public class ConstantVelocityTracker : VelocityTracker
+    {
+        /// <summary>
+        /// The velocity to return.
+        /// </summary>
+        [Serialized]
+        [field: DocumentedByXml]
+        public Vector3 Velocity { get; set; }
+
+        /// <summary>
+        /// The angular velocity to return.
+        /// </summary>
+        [Serialized]
+        [field: DocumentedByXml]
+        public Vector3 AngularVelocity { get; set; }
+
+        /// <summary>
+        /// Determines whether to extract the local property or the world property.
+        /// </summary>
+        [Serialized]
+        [field: DocumentedByXml]
+        public bool UseLocal { get; set; }
+
+        /// <inheritdoc />
+        protected override Vector3 DoGetAngularVelocity()
+        {
+            return UseLocal ? transform.localRotation * AngularVelocity : AngularVelocity;
```

Comment:
isn't this check pointless? why even have the UseLocal on this type?

Reply:
It is analogous to the Constant Force component’s world space force or relative force. 
Here we specify whether the velocity is world space or relative to self 
Also see above simulated throw use case.

Reply:
sorry, I read the line wrong, I didn't see the `transform.localRotation *`

this also needs a test for the class

Reply:
![image](https://user-images.githubusercontent.com/22952143/100381617-4fd9ec00-3054-11eb-9cdd-8dbed0458508.png)